src/Shared/ChunkingCookieManager/ChunkingCookieManager.cs

```diff
@@ -289,5 +306,32 @@ public void DeleteCookie(HttpContext context, string key, CookieOptions options)
             Expires = DateTimeOffset.UnixEpoch,
         });
     }
+
+    // Deletes unneeded cookie chunks, but not the primary cookie.
+#pragma warning disable CA1822 // Mark members as static - Shared src file
```

Comment:
Why would being a shared file change whether the method access instance members?

Reply:
Oh yeah 😆. I copied the suppression from the other method, but that one is needed because of the interface. This method can actually be static.